src/components/cluster/detail/ClusterDetailView.js

```diff
@@ -194,7 +194,7 @@ class ClusterDetailView extends React.Component {
     ) {
       return this.props.cluster.status.cluster.scaling.desiredCapacity;
     }
-    return null;
+    return 0; // if we return null no value is rendered in AWS v4 cluster view
```

Comment:
Can you also return `0` in the first `if` statement of the method, please?

Also, please replace all `!=` with `!==` 👍

Reply:
> Can you also return `0` in the first `if` statement of the method, please?

I am working wih AWS views now, and realize of this, so no, I won't cause I haven't check all use cases and maybe we will run into some issue.  Maybe later when I'm working with other providers. 

> Also, please replace all `!=` with `!==` 👍

👍